src/SFA.DAS.EAS.Portal.Client/SFA.DAS.EAS.Portal.Client.csproj

```diff
@@ -24,4 +25,9 @@
       <PackageReference Include="StructureMap" Version="4.7.1" />
     </ItemGroup>
 
+    <ItemGroup>
+      <ProjectReference Include="..\SFA.DAS.EAS.Portal.Types\SFA.DAS.EAS.Portal.Types.csproj" />
+      <ProjectReference Include="..\SFA.DAS.EAS.Portal\SFA.DAS.EAS.Portal.csproj" />
```

Comment:
The client was designed to *not* have a reference to the main portal project, for several reasons:
* it's not required
* information hiding
* keeping the size of the client package down

We should remove this reference to the portal project.

Reply:
Yep I agree but the Client referenced the Document which is in the Portal.  So maybe move those objects to the portal client.

Reply:
I think it should be OK to move `Document` and its interfaces into the client.

Reply:
👍